### PR TITLE
fix: banner logo sizing

### PR DESCRIPTION
**Pull request by montchr**
use the full logo at all screen sizes

### CODE REVIEW

**Comment by montchr** [comment]
all that said, the height of the site banner might look a little less extreme if the height of the Klein CTA banner also adjusted itself in proportion to the expansion of the site banner height... what do you think @bpizzle ?

**Comment by bpizzle** [comment]
this appears to be the 'correct' size for this logo to be visible, but the issue with this isn't how the Klein banner looks in comparison, to me it's the menu. it's too small vs the logo. i tested it with a larger font size (18px) and it still just looks bad floating there. i don't think adding more menu items would help.

**Comment by bpizzle** [comment]
what happens if we keep it this size, but put the text back to the left?

**Comment by bpizzle** [comment]
OK, I have some suggestions. Try this:

MAIN NAV:
nav item text:
`font-size: 16px;
font-weight: 600;
letter-spacing: .08em`

Logan Center logo: 85% of whatever the size is right now

KLEIN NAV:
Temple logo 35x35 px

text:
`.text-2xs {
font-size: .85rem;
}`

**Comment by bpizzle** [comment]
Actually the logo should be at 75% of whatever the size is now. Seems to be the same size as the one in the footer.

**Comment by montchr** [comment]
@bpizzle ready for review again.

also, check out #113

File: packages/tailwind-config/tailwind.config.js [hunk 1]
```diff
@@ -95,6 +95,15 @@ module.exports = {
         banner: '3 / 1',
         cinematic: '64 / 27', // a.k.a. "ultrawide" or "21:9" (a marketing term) <https://en.wikipedia.org/wiki/21:9_aspect_ratio>
       },
+      // Reference: // <https://tailwindcss.com/docs/customizing-spacing#default-spacing-scale>
+
+      // NOTE: keys *MUST* be strings -- `prettier` should not change this.
+      // TODO: this *might* be an upstream bug in `prettier-plugin-tailwind`
+      //       but first double-check that it's configured properly
+      // prettier-ignore
```
Comment (montchr) [inline packages/tailwind-config/tailwind.config.js:103]: might not be necessary actually

**Pull request merged**